lib/webtask.js

```diff
@@ -297,6 +303,15 @@ exports.issue_token = function (options /* req, claims, kid, keyset, store_code_
                     return cb(new Error('Error response from store_code_url when storing token. HTTP ' + res.statusCode + ': ' + (body || '<unknown>')));
                 cb();
             });
+        },
+        function (cb) {
+            if(new_claims.auth && !new_claims.jtn)
```

Comment:
The current expected format is (all optional):

```
{
  email: string or array of authed emails/domains,
  secret: base64 encoded secret for jwt verification,
  client_id: auth0 account client ID,
  auth0_domain: auth0 account domain
}
```

Looking at that I will group the auth0 stuff into one key so we can check they haven't sent incomplete info. We could also run a regex to check the `secret` looks like base64 if it's there.

If there are no options specified (just something truthy) then we allow all emails, but pass the auth0 token payload through to the webtask.